sdk/parents/azure-sdk-parent/pom.xml

```diff
@@ -43,6 +43,20 @@
         <updatePolicy>daily</updatePolicy>
       </snapshots>
     </repository>
+
+    <repository>
+      <id>repository.spring.milestone</id>
+      <name>Spring Milestone Repository</name>
+      <url>https://repo.spring.io/milestone</url>
+
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
```

Comment:
I'd want to put this repository behind a profile if we can, I don't think it'd be best to use Spring Milestone as a default repository.